docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+
+   .. list-table::
+      :header-rows: 1
+      :stub-columns: 1
+      :widths: 2 2 5
+
+      * - Application domain
+        - HPC application
+        - Description
+
+      * - Physics
+        - `Chroma <https://github.com/amd/InfinityHub-CI/tree/main/chroma/>`_
+        - Data-parallel programming constructs for lattice field theory -- lattice
+          quantum chromodynamics (QCD) in particular. It's used to simulate the
+          behavior of quarks and gluons. Chroma uses SciDAC QDP++ data-parallel
+          programming (in C++) that presents a single high-level code image to
+          the user, but can generate highly optimized code for many
+          architectural systems including single node workstations, multi and
+          many-core nodes, clusters of nodes via QMP, and classic vector
+          computers.
+
+      * -
+        - `Grid <https://github.com/amd/InfinityHub-CI/tree/main/grid/>`_
+        - Library for lattice QCD calculations that uses a high-level data
+          parallel approach and techniques to target multiple types of
+          parallelism. Grid currently supports MPI, OpenMP and short vector
+          parallelism. The code requires at least one AMD accelerator to run.
+
+      * -
+        - `MILC <https://github.com/amd/InfinityHub-CI/tree/main/milc/>`_
+        - Set of research codes developed by the MIMD Lattice Computation (MILC)
+          collaboration for doing simulations of four-dimensional SU(3) lattice
+          gauge theory on MIMD parallel machines scaling from single-processor
+          workstations to HPC systems.
+
+      * -
+        - `PIConGPU <https://github.com/amd/InfinityHub-CI/tree/main/picongpu>`_
+        - Particle-in-Cell code that runs on AMD accelerators. The PIC algorithm
+          is a central tool in plasma physics. It describes the dynamics of a
+          plasma by computing the motion of electrons and ions in the plasma
+          based on the Vlasov-Maxwell system of equations. 
+
+      * - Astrophysics
+        - `Cholla <https://github.com/amd/InfinityHub-CI/tree/main/cholla/>`_
+        - Static-mesh, GPU-native hydrodynamics simulation code that efficiently
+          runs high-resolution simulations on massively-parallel computers.
+          Cholla is written in a combination of C++, :doc:`HIP <hip:index>`, and
+          CUDA C, and requires at least one AMD accelerator to run.
+
+      * - Geophysics
+        - `SPECFEM3D Cartesian <https://github.com/amd/InfinityHub-CI/tree/main/specfem3d>`_
+        - Simulates acoustic (fluid), elastic (solid), coupled acoustic/elastic,
+          poroelastic or seismic wave propagation in any type of conforming mesh
+          of hexahedra (structured or otherwise). It can, for instance, model
+          seismic waves propagating in sedimentary basins or any other regional
+          geological model following earthquakes. It is also used for
+          non-destructive testing and for ocean acoustics.
```

Comment:
```suggestion
        - Simulates acoustic (fluid), elastic (solid), coupled acoustic/elastic, poroelastic, or seismic wave propagation in any conforming mesh of hexahedra (structured or otherwise). . It can, for instance, model
          seismic waves propagating in sedimentary basins or any other regional
          geological model following earthquakes. It is also used for
          non-destructive testing and for ocean acoustics.
```